src/agent/Cargo.lock

```diff
@@ -1202,6 +1202,7 @@ dependencies = [
  "oci",
  "once_cell",
  "rand",
+ "safe-path",
```

Comment:
How Cargo.lock can be changed with Cargo.toml unchanged?

Reply:
Thx @Tim-Zhang fixed